code/game/machinery/_machines_base/stock_parts/access_lock.dm

```diff
@@ -15,6 +18,11 @@
 				return SPAN_WARNING("\The [lock] flashes red! You lack the access to unlock this.")
 	return ..()
 
+/obj/item/stock_parts/access_lock/get_req_access()
+	if(locked)
+		return ..()
+	return list()
```

Comment:
I think this isn't needed, but if you do include it you should change the proc just above.

Reply:
Wasn't sure if access handling handles getting nulls instead of accesses

Reply:
Changed the check to check if we're in the machine instead, and returns null here.